### PR TITLE
Add globe release test with `setData` calls and map aligned symbol

### PR DESCRIPTION
**Pull request by avpeery**
This release testing page for globe view replaces the `animate a point` testing page with the flight path testing page. The flight path testing page addresses the same major point as `animate a point` (multiple `setData` calls) and had previously exhibited issues when used in globe view (now fixed) in regards to map-aligned symbols (see: #11775).

## Launch Checklist

 - [x] briefly describe the changes in this PR
 - [x] manually test the debug page

### CODE REVIEW

File: debug/animate-point-along-route.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            'id': 'point',
+            'source': 'point',
+            'type': 'symbol',
+            'layout': {
+                // This icon is a part of the Mapbox Streets style.
+                // To view all images available in a Mapbox style, open
+                // the style in Mapbox Studio and click the "Images" tab.
+                // To add a new image to the style at runtime see
+                // https://docs.mapbox.com/mapbox-gl-js/example/add-image/
+                'icon-image': 'airport-15',
```
Comment (SnailBones) [inline debug/animate-point-along-route.html:151]: Perhaps a larger icon to make it easier to verify draping behavior on globe?

```suggestion
                'icon-image': 'airport-15',
                'icon-size':3,
```

**Review by SnailBones — COMMENTED** [review]
Thoughts on consolidating this with another test (Maybe the image test in https://github.com/mapbox/mapbox-gl-js/pull/11881) to keep the number of pages down?

**Comment by avpeery** [comment]
> Thoughts on consolidating this with another test (Maybe the image test in #11881) to keep the number of pages down?

So this page is replacing the previous release testing page that also testing for setData... (this one)[https://docs.mapbox.com/mapbox-gl-js/example/animate-point-along-line/]. So it is not adding a new page, and same with the image source test - it's updating the current release testing image-source to include globe. So no extra tests are added to the total amount of release tests (aside from the atmosphere playground in a different PR). My opinion is to keep these as separate tests to make the release test obvious what to look for to the tester in these cases. Thanks for the review!

**Review by SnailBones — APPROVED** [review]
> So this page is replacing the previous release testing page that also tested for setData performance.

Whoops, I missed that! Sounds good to me! 👍

**Review by karimnaaji — APPROVED** [review]
👍 LGTM

**Pull request merged**